tests/bin/distrib-name-dune-project/run.t

```diff
@@ -0,0 +1,32 @@
+When a package name is inferred from dune-project, it should only use the
```

Comment:
Ideally I would have preferred a unit test for this but we'll settle for a cram test.

Reply:
I can do that

Reply:
done!